burf-base/_tables.scss

```diff
@@ -156,8 +102,6 @@ table {
 	}
 
 	thead {
-		background: $table-head-bg;
```

Comment:
I'm hesitant to remove some of these for non-block tables at least. Could we add custom properties here? Maybe Crystal's PR is already doing that?